test/models/booking_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class BookingTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end

```

Comment:
You can delete this marked file you can also use this command aswell for another time

`rails generate model 'MODEL' --no-test-framword --no-helpers`